### ISSUE TITLE
bug: custom token name is not working with jwt token proovider.

### ISSUE THREAD

**Issue by bleonard252** [greenpau/caddy-auth-portal#50]
After "detected cookie-based redirect", no other errors are being logged for this in Caddy (debug is on), but both Firefox and Chrome redirect /auth/settings to /auth. I can confirm sign-in is successful because / shows my index.html instead of redirecting to /auth, which it does after going to /auth/logout. /auth/whoami doesn't work either. Going to /auth/blblblbl (among other things) does give the "not found" page.

My Caddyfile (with redactions and removal of comments) looks like:
```caddyfile
{
   debug
}
http://127.0.0.1:8080 {

   # Auth config
   route /auth* {
      auth_portal {
         path /auth
         backends {
            local_backend {
               method local
               path /data/jwtusers.json
               realm local
            }
         }
         jwt {
            token_name usertoken
            token_secret [unique UUID 1]
            token_issuer [unique UUID 2]
         }
      }
   }

   route {
      jwt {
         primary yes
         auth_url /auth
         trusted_tokens {
            static_secret {
               token_name usertoken
               token_secret [unique UUID 1]
               token_issuer [unique UUID 2]
            }
         }
         allow roles admin
         allow roles superadmin
         allow roles admin editor viewer
      }
      root * /data/htdocs
      file_server
   }
}
```

In case you're curious, the Dockerfile I'm using looks like:
```dockerfile
FROM caddy:builder AS builder

RUN xcaddy build \
    --with github.com/greenpau/caddy-auth-jwt \
    --with github.com/greenpau/caddy-auth-portal \
    --with github.com/hairyhenderson/caddy-teapot-module@v0.0.3-0

FROM caddy:latest

COPY --from=builder /usr/bin/caddy /usr/bin/caddy
```
and docker-compose.yml:
```yaml
version: '3.8'
services:
  web:
    build: .
    network_mode: host
    volumes:
      - ./data:/data
      - ./config:/config
      - ./Caddyfile:/etc/caddy/Caddyfile
```

**Comment by greenpau** [comment]
@bleonard252 , are you using the latest plugin version? What is the version you use?

**Comment by greenpau** [comment]
@bleonard252 , also, did you read about “cookie_domain”?

**Comment by bleonard252** [comment]
(That was fast.)
I'm pretty sure I'm using the latest, maybe it's the release version. It's whatever xcaddy gave me with that command.

**Comment by greenpau** [comment]
@bleonard252 please do “@latest” in the xcaddy command for auth plugins

**Comment by bleonard252** [comment]
Will try that. I don't know where to set cookie domain, is that in the caddy docs somewhere?

**Comment by greenpau** [comment]
@bleonard252 ,domain_cookie does not apply here because you are hosting and serving on the same node.

**Comment by greenpau** [comment]
@bleonard252 , the issue here is your “root” directive

**Comment by greenpau** [comment]
@bleonard252 , you need to specify “root” inside your file_server. That what messes it all up.

**Comment by bleonard252** [comment]
I'll try that, thank you

**Comment by greenpau** [comment]
Also, do not do “http://127.0.0.1:8080” ... it is just 127.0.0.1 ... And you could have multiple names and it addresses separated by comma. The port is defined at the top. See here https://github.com/greenpau/caddy-auth-portal/blob/main/assets/conf/local/Caddyfile

**Comment by greenpau** [comment]
@bleonard252 , regarding “root” , please how it is defined here https://caddyserver.com/docs/caddyfile/directives/file_server

**Comment by bleonard252** [comment]
For some reason, changing the token name to "access_token" seemed to fix it.

**Comment by greenpau** [comment]
@bleonard252 , thank you for letting me know! It could be bug. Will check it out later.

**Comment by greenpau** [comment]
@bleonard252 , it was a bug. Thanks again for reporting it! The fix is available in [v1.2.6](https://github.com/greenpau/caddy-auth-portal/releases/tag/v1.2.6).

Feel free to give this plugin a star and promote the plugins (portal, jwt) on the social platforms you use!
